e2e_tests/dedicated_admin_rolebindings.py

```diff
@@ -32,10 +32,11 @@ def test_cluster(cluster, oc_map, pattern):
 
 
 @defer
-def run(thread_pool_size=10, defer=None):
+def run(thread_pool_size=10, dry_run=False, defer=None):
     oc_map = tb.get_oc_map(QONTRACT_E2E_TEST)
     defer(oc_map.cleanup)
     pattern = tb.get_namespaces_pattern()
     threaded.run(test_cluster, oc_map.clusters(), thread_pool_size,
                  oc_map=oc_map,
-                 pattern=pattern)
+                 pattern=pattern,
+                 dry_run=dry_run)
```

Comment:
I don't think you need to pass this option to `test_cluster` if there's nothing being done there with it